Sources/JWTKit/X5CVerifier.swift

```diff
@@ -121,13 +121,27 @@ public struct X5CVerifier: Sendable {
             return data
         }
 
-        // Setup an untrusted chain using all the certificates in the x5c.
+        // Setup an untrusted chain using all the certificates in the x5c
         let untrustedChain = try CertificateStore(certificateData.map {
             try Certificate(derEncoded: [UInt8]($0))
         })
 
+        var date = Date()
```

Comment:
Set as a let and use an else block down below instead of overwriting it if we need to